### DISCUSSION TITLE
STM32 Uart log never take effect

### DISCUSSION THREAD

**Discussion by Jeepgoing** [zephyrproject-rtos/zephyr#43137] [Q&A]
https://github.com/zephyrproject-rtos/zephyr/blob/5fab2efc791ad5329d49ff2e0dadd76b4471dfa3/drivers/serial/uart_stm32.c#L39

maybe change to
`LOG_MODULE_REGISTER(uart_stm32, LOG_LEVEL_ERR);`
like https://github.com/zephyrproject-rtos/zephyr/blob/5fab2efc791ad5329d49ff2e0dadd76b4471dfa3/drivers/serial/uart_nrfx_uarte.c#L19

**Comment by yashi** [comment]
The [documentation](https://docs.zephyrproject.org/apidoc/latest/group__log__api.html#ga2404243df68fb6e51129d1c7ecc5ca45) says:

> [`LOG_MODULE_REGISTER()`] accepts one or two parameters:
>  - module name
>  - optional log level. If not provided then default log level is used in the file.
